app/views/curation_concerns/generic_works/_browse_everything.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render 'curation_concerns/file_sets/upload/agreement' %>
+<div class="alert alert-success">
+  <%=  t("sufia.upload.cloud_timeout_message", contact_href: link_to(t("sufia.upload.alert.contact_href_text"), sufia.contact_form_index_path)) %>
+</div>
+<div class="well">
+  <%= form_tag(@presenter ? main_app.curation_concerns_file_sets_path(parent_id: @presenter.id) : sufia.curation_concerns_file_sets_path, id: 'browse_everything_form', method: 'post') do %>
+      <%= render 'curation_concerns/file_sets/upload/tos_checkbox' %>
+      <%= hidden_field_tag(:upload_set_id, @upload_set_id) %>
+      <%= button_tag(t('sufia.upload.browse_everything.browse_files_button'), type: 'button', class: 'btn btn-lg btn-success', id: "browse-btn",
+        'data-toggle' => 'browse-everything', 'data-route' => browse_everything_engine.root_path,
+        'data-target' => '#browse_everything_form' ) %>
+      <%= button_tag("Upload cloud files", type: 'submit', class: 'activate-submit btn btn-lg btn-primary', id: "submit-btn") %>
+  <% end %>
+
+  <p id="status">0 items selected</p>
+</div>
+<script>
```

Comment:
TODO: lookup script tag and asset pipeline for rails 4.  Specifically, there might be an optimization for this, or it might not be that big of a deal now.